src/components/StyleGuide/Forms.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import { getCategory } from "../../utils/componentCategories";
+import getCSSVariable from "../../helpers/getCSSVariable";
+import Card from "../Card/Card";
+import List from "../List/List";
+import { ListTypes } from "../List/ListTypes";
+
+<Meta
+  title={getCategory("Forms")}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
```

Comment:
I don't think you need this because it doesn't show up for documentation pages.

Reply:
Yes, that's true.  It's now removed.